tests/integrational/ContenttypeTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests\integrational;
```

Comment:
Actually, I took a look at the test-namespaces in [transfer/transfer TransferTestCase](https://github.com/transfer-framework/transfer/blob/1.0/tests/functional/TransferTestCase.php).
Anyway, should I also structure it in the suggested namespace path? I must have misunderstood our talk earlier this week.